lightyear/src/client/prediction/plugin.rs

```diff
@@ -199,6 +200,17 @@ pub fn add_non_networked_rollback_systems<C: Component + PartialEq + Clone>(app:
     );
 }
 
+pub fn add_resource_rollback_systems<R: Resource + Clone>(app: &mut App) {
```

Comment:
maybe we also need an observer that adds the `ResourceHistory` resource when the resource first gets added

Reply:
oh i see you insert it directly in the protocol function, I think that works too!